### DISCUSSION TITLE
Using this action internally in our organization- unable to resolve the actions used by this action.

### DISCUSSION THREAD

**Discussion by SuganJoe** [tj-actions/changed-files#1079] [Q&A] [answered]
when trying to use this action internally, unable to resolve the actions used by this action. 
Note: this action is enabled to use internally.
Error:
Unable to resolve action `tj-actions/glob@v16.14`, repository not found on this server. If you want to use this action from [GitHub.com](http://github.com/), see the following documentation: https://docs.github.com/en/enterprise/admin/github-actions/managing-access-to-actions-from-githubcom. 
Unable to resolve action `tj-actions/json2file@v1.4.0`, repository not found on this server. If you want to use this action from [GitHub.com](http://github.com/), see the following documentation: https://docs.github.com/en/enterprise/admin/github-actions/managing-access-to-actions-from-githubcom

**Comment by SuganJoe** [comment]
Is there a possibility of preventing hard references in this action (or)  are we supposed to have the other actions enabled to use internally as well?

**Reply by jackton1** [reply, marked as answer]
Yes, you’ll need to have all dependent actions enabled.

See: https://docs.github.com/en/enterprise-server@3.8/admin/github-actions/managing-access-to-actions-from-githubcom/about-using-actions-in-your-enterprise#configuring-access-to-actions-on-githubcom
